docker/install/ubuntu_install_vela.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+set -u
+set -o pipefail
+
+pip3 install -U setuptools
+pip3 install ethos-u-vela==2.1.1
```

Comment:
could you add this dependency to `python/gen_requirements.py` as an `ethosu` extra?

also, our policy for python deps here has generally been to not specify a version (this way as containers are rebuilt, we add in the work needed to maintain compatibility with the latest version). is there a functional reason why we must have `2.1.1` here?

Reply:
I've added Vela as an ethosu extra to python/gen_requirements.py now.

Vela 2.1.1 is the version that is currently supported and we know we have some issues with later versions which need fixes that are being worked on. But until that happens, we really do need this pinned at 2.1.1.

Reply:
ok great. is there any further context you can provide publicly (or link to a tracking bug) describing issues with the 3.0.0 release or tracking the effort to be compatible with latest ethos-u-vela? i'd like to place a pointer in the code (in comment) so that interested readers can follow-up if desired

Reply:
Hi Andrew, unfortunately we don't have a publicly available bug for tracking this issue.

Reply:
thanks @grant-arm!